en/docs/administer/key-managers/configure-okta-connector.md

```diff
@@ -329,6 +329,9 @@ Follow the instructions below to configure Okta as a third-party Key Manager:
           </tr>
         </table>
 
+!!! tip
```

Comment:
It is better to say that `well-known-url` can be found under the parameter `Metadata URI` in the settings related to the Authorization server